Utilities/build-script-helper.py

```diff
@@ -68,13 +68,16 @@ def swiftpm_bin_path(swift_exec: str, swiftpm_args: List[str], additional_env: D
     return check_output(cmd, additional_env=additional_env, capture_stderr=False, verbose=verbose).strip()
 
 
-def get_build_target(swift_exec: str, args: argparse.Namespace) -> str:
+def get_build_target(swift_exec: str, args: argparse.Namespace, cross_compile: bool = False) -> str:
```

Comment:
Since we are currently passing all arguments via the `args` parameter, could we also avoid passing `cross_comile` and instead check whether `args.corss_compile_config` is set inside this function?

Reply:
The reason it is done this way is because I copied it over from how [I did it for SPM a couple years ago](https://github.com/apple/swift-package-manager/commit/38df7c37e7ff705c6f48544a28d3d7a4839e290f#diff-fdc89d912e967e4b700e1bb41e08552bb416b5c5551bbae5b1ccac134db6c3e5R284). This is more general and allows the caller to manually enable checking for cross-compilation triples only when needed, which is used by that SPM bootstrap script.

While you're right that such generality is not needed for this script right now, if you think it might be later, better to keep it this way.

Reply:
OK, fair enough 👍 I think you also added it to SourceKit-LSP back then and I removed it when I refactored `build-script-helper.py` because we didn’t need it.